packages/esm-ward-app/src/hooks/useInpatientRequest.ts

```diff
@@ -1,10 +1,9 @@
-import { openmrsFetch, restBaseUrl } from '@openmrs/esm-framework';
+import { openmrsFetch } from '@openmrs/esm-framework';
 import useSWR from 'swr';
 import type { InpatientRequest } from '../types';
 
-//TODO:The api is still in implementation stage and this is just a placeholder for the api
 export function useInpatientRequest(locationUuid: string) {
-  const apiUrl = `${restBaseUrl}/emrapi/inpatient/admissionRequests?admissionLocation=${locationUuid}`;
+  const apiUrl = `/ws/rest/emrapi/inpatient/admissionRequests?admissionLocation=${locationUuid}`;
```

Comment:
Does this imply that we have a bug in emrapi endpoint that it is not using the appropriate base url prefix?  If so, we should fix it there and put this back.

Reply:
Yes. Just filed https://openmrs.atlassian.net/browse/EA-194. Most (all?) emrapi endpoints have the same problem so this could be relatively disruptive to fix.